homeassistant/components/electric_kiwi/api/README.md

```diff
@@ -0,0 +1,16 @@
+# electrickiwi.api
```

Comment:
From my experience, it is easier to move the entire api code in a pypi package and referr to it in the manifest.yaml as requirement.  
This also removes non-home assistant specific code from the core codebase (beside the advantages, api specific code is not wanted in the code base) 
see https://developers.home-assistant.io/docs/api_lib_index

Reply:
I will need to investigate that as its a common pattern I took out of other components including a google one. It also requires `from homeassistant.helpers import config_entry_oauth2_flow` The api.py is only for oauth a session.
I followed
https://developers.home-assistant.io/docs/api_lib_auth

Reply:
I should add I did write an entire API client externally that is referenced in manifest.yaml

Reply:
on a different note the issues with mypy is resolved now.

Reply:
@Lurchi70 I don't think what I've done is wrong, I think you may be assuming I've got the API in the PR which I don't. Welcome feedback. thanks.

Reply:
The comment is on a super old commit (which does have the API code), that might be where the confusion is coming from.

Reply:
Correct. Sorry for the confusion. 
Hae seen that there was a lot of progres later.